releases/2020-11/js.md

```diff
@@ -34,12 +37,54 @@ If you have a bug or feature request for one of the libraries, please post an is
 
 ## Release highlights
 
-### _Package name_
+---
+
+=== COPY THIS AND ADD THE INFORMATION OF YOUR PACKAGE: ===
+
+Keep in mind that:
+
+- Including the package name in the headers makes the URL links work for multiple packages.
+- The format of this file will be cleaned up once all of your proposals are in.
+
+---
+
+### _Package name_ [Changelog](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/<service-folder>/<package-folder>/CHANGELOG.md)
+
+(leave blank)
+
+##### Breaking Changes on _Package name_
+
+- _Add one or more, or remove the "Breaking Changes on ..." entire section._
+
+##### New Features on _Package name_
 
-- Major changes only!
+- _Add one or more, or remove the "New Features on ..." section._
+
+##### Major Fixes on _Package name_
+
+- _Add one or more, or remove the "Major Fixes on ..." section._
+
```

Comment:
```suggestion
### Azure Tables

#### @azure/data-tables [Changelog](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/tables/data-tables/CHANGELOG.md)

We're releasing a new preview of our Azure Tables library. This update provides more idiomatic names to the system properties for `odata.etag` and `Timestamp`.

##### Major fixes on @azure/data-tables@1.0.0-beta3
Renamed system properties `odata.etag` and `Timestamp` to provide more idiomatic property names. Queried entities now get the properties `etag` and `timestamp` instead of `odata.etag` and `Timestamp`,
```

Reply:
I added it below! Thank you!